railties/lib/rails/test_help.rb

```diff
@@ -14,10 +14,12 @@
 
 if defined?(ActiveRecord::Base)
   require "active_record/testing/query_assertions"
+  require "active_support/testing/instrumentation_assertions"
   ActiveSupport.on_load(:active_support_test_case) do
     include ActiveRecord::TestDatabases
     include ActiveRecord::TestFixtures
     include ActiveRecord::Assertions::QueryAssertions
+    include ActiveSupport::Testing::InstrumentationAssertions
```

Comment:
Since this is nested within an `defined?(ActiveRecord::Base)`, it will only work when Active Record is available.

Active Support can be used independently from other parts of the framework.

If making the module globally available is the goal, this should be moved to [ActiveSupport::TestCase](https://github.com/rails/rails/blob/fa152fba87f3a11790783207e6e84e58d345c012/activesupport/lib/active_support/test_case.rb#L144-L153).

If it's opt-in, it should be required in [lib/active_support/test_case.rb](https://github.com/rails/rails/blob/fa152fba87f3a11790783207e6e84e58d345c012/activesupport/lib/active_support/test_case.rb#L3-L18) and included ad-hoc test-by test in the application code.

I can't really weigh in on which it should be, I defer that to Core.

Reply:
Good eye. Moving to `ActiveSupport::TestCase`. Let's see what folks say.

Reply:
Good eye. Will move to `ActiveSupport::TestCase`. Let's see what folks say.